doc/manual/types.rst

```diff
@@ -120,8 +120,8 @@ using :func:`convert`:
 .. doctest:: foo-func
 
     julia> function foo()
-             x::Int8 = 100
-             x
+               x::Int8 = 100
+               x
            end
```

Comment:
That's the style guideline for base code, so I think picking something for consistency in doc code examples makes sense?